src/robot/libraries/String.py

```diff
@@ -346,6 +352,28 @@ def remove_string_using_regexp(self, string, *patterns):
         for pattern in patterns:
             string = self.replace_string_using_regexp(string, pattern, '')
         return string
+		
+    def remove_lines_containing_string(self, string, pattern, case_insensitive=False):
+        """Remove lines of the given ``string`` that contain the ``pattern``.
+
+        The ``pattern`` is always considered to be a normal string and a
+        line matches if the ``pattern`` is found anywhere in it. By
+        default the match is case-sensitive, but setting
+        ``case_insensitive`` to any value makes it case-insensitive.
+
+        Non matching lines are returned as one string catenated back together with
+        newlines. Possible trailing newline is never returned. The
+        number of non matching lines is automatically logged.
+
+		This does exact opposite functionality of Get Lines Containing String
+        """
+        if case_insensitive:
+            pattern = pattern.lower()
+            contains = lambda line: pattern in line.lower()
+        else:
+            contains = lambda line: pattern in line
+        return self._get_non_matching_lines(string, contains)
+
 
```

Comment:
Extra empty line added here. We only have one between methods as PEP-8 recommends.